image/store/distribution.go

```diff
@@ -41,7 +41,8 @@ func (fs *filesystem) LoadDistributionMetadata(layerID LayerID) (map[string]dige
 	)
 	distributionMetadataFile, err := os.Open(filepath.Join(layerDBPath, "distribution_layer_digest"))
 	if err != nil {
-		return res, nil
+		//lint:ignore nilerr https://github.com/alibaba/sealer/issues/610
```

Comment:
I think we need return error here,not ignore it~

Reply:
@justadogistaken double check this~

Reply:
There would be some problems, distribution_layer_digest does not exist for all layers, which will be generated by "pull"/"push", but "build". If we return err here, layer can't be loaded.

Reply:
Maybe we should return error here, it's just a low-level func. But we need to ignore the error at the caller side

Reply:
`ignore the error at the caller side`, agree~

Reply:
so it's my turn to take next pr, right.
which steps should i choose:
- create a new branch from main and modify the codes 
- modify the codes based on the last pr branch
@fanux

Reply:
> so it's my turn to take next pr, right.
> which steps should i choose:
> 
> * create a new branch from main and modify the codes
> * modify the codes based on the last pr branch
>   @fanux

Just modify this branch and push~

Reply:
👌 , I got it wrong